docs/QSG/docker.md

```diff
@@ -6,16 +6,16 @@ using DAOS as backend storage.  This small cluster is composed of the following
 - The `daos-server` node running a DAOS server daemon managing data storage devices such as SCM or
```

Comment:
In the PR description, you mentioned needing a string when on the SDP cloud server, however, there is no sample command for doing such in the QSG. What's the chance an end-user might encounter the problem and an example would be in order?

Reply:
I added a new commit explaining how to do it: https://github.com/daos-stack/daos/commit/9ebf78cab1d69a60dde34f7e90c581c8e57863fb

I also updated the first two commit messages to respect the commit name guidelines of DAOS.